### ISSUE TITLE
Load balancing algorithm used?

### ISSUE THREAD

**Issue by dstiliadis** [kinvolk/service-mesh-benchmark#8]
I was wondering if anyone has played with the Load Balancing algorithms used in the benchmark. I believe Linkerd comes up with default Power of Two Choices. The corresponding one in Istio is the Weighted Least Request, but if I am not mistaken uses Round Robin by default. I couldn't find any configuration for those in the repo. Probably I am missing something.

The Load Balancing algorithm can have a dramatic effect in the 99-percentile latency observations provided CPU is not saturated, in which case the CPU effects take over.

In other words, it is not clear from the results if the effects are because of CPU utilization and saturation or load balancing algorithm selection that in turn affects CPU saturation in upstream nodes.

Thanks in advance for any clarifications.
